### PR TITLE
feat: support drift detection

### PR DESCRIPTION
**Pull request by suzuki-shunsuke**
- https://github.com/suzuki-shunsuke/tfaction/issues/851

### CODE REVIEW

File: tfmigrate-apply/action.yaml [hunk 1]
```diff
@@ -9,17 +9,24 @@ inputs:
...
+
 runs:
   using: composite
   steps:
     - uses: suzuki-shunsuke/tfaction/get-global-config@main
       id: global-config
     - uses: suzuki-shunsuke/tfaction/get-target-config@main
       id: target-config
+    - uses: suzuki-shunsuke/tfaction/get-global-config@main
+      id: global-config
```
Comment (yuya-takeyama) [inline tfmigrate-apply/action.yaml:21]: `global-config` is duplicated.

ref:

* https://github.com/suzuki-shunsuke/tfaction/issues/906

**Comment by suzuki-shunsuke** [comment]
Released v0.6.0-3.
https://github.com/suzuki-shunsuke/tfaction/actions/runs/4829407931
https://github.com/suzuki-shunsuke/tfaction/releases/tag/v0.6.0-3

**Comment by suzuki-shunsuke** [comment]
v0.6.0-4
https://github.com/suzuki-shunsuke/tfaction/releases/tag/v0.6.0-4
[7a3cf78](https://github.com/suzuki-shunsuke/tfaction/pull/876/commits/7a3cf7802ed998df4a70a602d82704c2233648fc)

**Comment by suzuki-shunsuke** [comment]
v0.6.0-5
https://github.com/suzuki-shunsuke/tfaction/releases/tag/v0.6.0-5

**Comment by suzuki-shunsuke** [comment]
v0.6.0-6
https://github.com/suzuki-shunsuke/tfaction/releases/tag/v0.6.0-6

**Pull request merged**